.github/workflows/master_post_build_convoy.yml

```diff
@@ -153,6 +155,12 @@ jobs:
                 exit(1)
               print("POST request Complete")
 
+            def get_software_def_url(payload, API_URL):
+              if STATUS == 'SUCCESSFUL':
```

Comment:
Where is `STATUS` getting initialised?

Reply:
Also why uppercase?

Reply:
> Where is STATUS getting initialised?

here https://github.com/TessellDevelopment/convoy-ci/pull/362/files#diff-bb4c9bf690a590a7925bedd00654d65a600692048c6270897c58037d1e2196bdR172